Kagegami High/Kagegami.json

```diff
@@ -0,0 +1,9 @@
+{
+	"html": "kagegami.html",
+	"css": "kagegami.css",
```

Comment:
The name of the html & css files need to exactly match the files included. Capitalization included. You need to either uppercase the first letters or renamed your files to match this name.

Reply:
No a problem. It's a common mistake. I do it myself. ;)